doc/source/admin/galaxy_options.rst

```diff
@@ -1376,6 +1376,16 @@
 :Type: bool
 
 
+~~~~~~~~~~~~~~~~
+``aws_estimate``
+~~~~~~~~~~~~~~~~
+
+:Description:
+    Enable AWS estimate.
```

Comment:
a little bit more description would be nice. Something along these lines.
This flag enables an AWS cost estimate for every job based on their runtime matrices. For this, the CPU and memory usage and the runtime is takes and mapped against AWS pricing table (path ...). Please note, that those numbers
are only estimates.

Reply:
@bgruening Thanks, it is a good idea, I will do.